pkg/lib/cockpit-components-listing-panel.jsx

```diff
@@ -26,6 +26,7 @@ import './cockpit-components-listing-panel.scss';
  *     - name tab name (has to be unique in the entry, used as react key)
  *     - renderer react component
  *     - data render data passed to the tab renderer
+ *     - hide flag whetever tab link and it's content should be shown
```

Comment:
This sentence has some grammar issues. Why do you need the hide flag? What if you just don't pass it?

Reply:
Fixed